packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -505,6 +506,10 @@ class _HeroFlightManifest {
     return '_HeroFlightManifest($type tag: $tag from route: ${fromRoute.settings} '
         'to route: ${toRoute.settings} with hero: $fromHero to $toHero)${isValid ? '' : ', INVALID'}';
   }
+
+  void dispose() {
+    _animation?.dispose();
```

Comment:
Can you mark it with `@mustCallSuper`, in case someone will decide to override it?

Reply:
Sure ! I added it in [fix: Add mustCallSuper](https://github.com/flutter/flutter/pull/147303/commits/db7a3d321784901e055f0404af61e2b600cd6eb6)